client/src/app/Main.res

```diff
@@ -2226,10 +2277,10 @@ let subscriptions = (m: model): Tea.Sub.t<msg> => {
 
   let visibility = list{
     BrowserSubscriptions.Window.OnFocusChange.listen(~key="window_on_focus_change", v =>
-      if v {
+      if !v {
```

Comment:
I'll double-check this, but somehow, I found that the existing tracking of `PageVisibility` was _backwards_. I'm not sure what has been reliant on that field so far (has something been secretly broken, or am I wrong in my assessment here?)

Reply:
This doesn't read correctly as-is. Something needs to change.

Reply:
given `appsupport` is currently going through changes, I'll resolve this by updating the ReScript listener to expect a `hidden` bool rather than a `visible` bool

Reply:
this is done